content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -137,20 +153,55 @@ use, see `man 7 user_namespaces`.
 
 ## Set up a node to support user namespaces
 
-It is recommended that the host's files and host's processes use UIDs/GIDs in
-the range of 0-65535.
-
-The kubelet will assign UIDs/GIDs higher than that to pods. Therefore, to
-guarantee as much isolation as possible, the UIDs/GIDs used by the host's files
-and host's processes should be in the range 0-65535.
+By default, the kubelet assumes host's files and host's processes use UIDs/GIDs
+in the range of 0-65535. This is the case in most Linux distributions with the
+default installation. The kubelet will assign UIDs/GIDs higher than that to
+pods, so there is no overlap between UID/GIDs used by host's files and processes
+and the ones used by pods.
 
 Note that this recommendation is important to mitigate the impact of CVEs like
 [CVE-2021-25741][CVE-2021-25741], where a pod can potentially read arbitrary
-files in the hosts. If the UIDs/GIDs of the pod and the host don't overlap, it
+files in the host. If the UIDs/GIDs of the pod and the host don't overlap, it
 is limited what a pod would be able to do: the pod UID/GID won't match the
 host's file owner/group.
 
+The kubelet can use a custom range for pods too. To configure a custom range,
+the node needs to have:
+
+ * A user `kubelet` in the system
+ * The binary `getsubids` installed (part of the [shadow-utils][shadow-utils] package)
+ * A configuration of subordinate UIDs/GIDs for the `kubelet` user (see `man 5
+   subuid` and `man 5 subgid`)
+
+The `kubelet` user will only be used to gather the UID/GIDs range configuration,
+but the user the kubelet is executed as won't be changed by this.
+
+There are some requirements to configure the subordinate IDs:
+
+ * The subordinate user ID must be a multiple of 65536 and greater than or equal
+   to it too. In other words, the first range 0-65535 can't be used for pods, as
+   that will circumvent some security benefits of user namespaces.
+
+ * The subordinate ID count must be a multiple of 65536
+
+ * The subordinate ID count must be at least `65536 * <maxPods>` where `<maxPods>`
+   is the maximum number of pods that can run on the node.
+
+ * The configuration for UIDs and GIDs for the `kubelet` user must be identical
+
+For example, this can be set in `/etc/subuid` and `/etc/subgid`:
```

Comment:
```suggestion
For example, you could define `/etc/subuid` and `/etc/subgid` to both have contents
similar to:
```

Reply:
Added that. But changed the wording to:

```
For example, you could define `/etc/subuid` and `/etc/subgid` to both have
thsese entries for the `kubelet` user:
```

Before we were saying you could set that, but you were saying the file can be like this. Let's say what we were saying, the kubelet entry can look like this (we don't care about the rest of the file).